docs/index.adoc

```diff
@@ -23,7 +23,7 @@ Given a _start_ object, korrel8r searches for a chain of related rules, and find
 Korrel8r can be run as a plain executable, outside the cluster, and connect to the API server and stores inside the cluster. It can also run as a deployed service inside the cluster.
 
 [id="index-architecture"]
-== Korrel8r architecture
+== Korrel8r Architecture
```

Comment:
```suggestion
== Korrel8r architecture
```
Use sentence case for headings 😉

Reply:
I did not know that. There are many headings for you to correct :stuck_out_tongue_winking_eye: